src/ASBNApp.DataAPI/ASBNApp.DataAPI.http

```diff
@@ -0,0 +1,6 @@
+@ASBNApp.DataAPI_HostAddress = http://localhost:5030
+
+GET {{ASBNApp.DataAPI_HostAddress}}/weatherforecast/
+Accept: application/json
+
+###
```

Comment:
maybe add here usefull examples so others can fast explorer your api :) maybe use the endpoints explorer in your visual studio to generate the content of that file